episodes/formative-assessment.md

```diff
@@ -155,6 +174,63 @@ they might have misconceptions that cause them to arrive at an answer that is in
 but not represented in the MCQ options.
 
 ::::::::::::::::::::::::::::::::::::::::
+                    
+### Example MCQs
+
+Thinking back of the misconceptions of how values are assigned, referenced and updated in programming languages, 
+here is an example MCQ that can probe learners for such misconceptions. 
+
+_MCQ: Look at the following 3 assignment statements in Python._
+
+```python
+    n = 300
+    m = n
+    n = -100
+```
+
+_What is the result at the end of the above assignments?_
+
+```python
+    a. n = 300 and m = 300
+    b. n = -100 and m = 300
+    c. n = -100 and m = -100
+    d. n = 300 and m = -100
+```
+
+The correct answer is b., while each of the plausible distractors identify a different misconception:
+
+a) they understand the value held by `n` is now also held by `m` but do not understand the value of 
+`n` has been updated since the initial assignment (e.g. treating it as a constant).
+c) they do not understand `n` and `m` are separate variables/containers for values (which can hold the same values) 
+but rather assume that `m` is referencing variable `n` instead of copying its value at the time of assignment (e.g. behaving as a reference to another cell in a spreadsheet).
+d) looks like a mix of misconceptions a. and c.
```

Comment:
Maybe a better 3rd diagnostic answer (for d) would be something like n = 200, m=300 - this could be if they think that the reassignment for n actually subtracts 100 from the original value as their misconception.

Reply:
That's a good one - will add.

Reply:
Fixed via [d6c2c30](https://github.com/carpentries/lesson-development-training/commit/d6c2c30117e227af182c4b4b51e7e41775c2dbca)